### DISCUSSION TITLE
Slingboxes have stopped trying to "phone home"

### DISCUSSION THREAD

**Discussion by mdehrlich** [GerryDazoo/Slinger#509] [General]
I just checked my router security log.  I have my slingboxes blocked for any internet traffic.  The router logs all blocked attempts.  After a month of repeated (several per minute) attempts by them to reach out to the sling servers, they stopped.  Dec 21 was the last attempt.  Boxes continue to work great with slinger.

**Comment by regmar505** [comment]
I wonder if you cycle the power of your Slingbox whether attempts to reach out the sling servers will restart.

**Reply by mdehrlich** [reply]
Not going to force it.  If power goes down, I'll check to see if it restarts the process.

**Reply by sp222222** [reply]
keep us posted... I restart my hardware once a week sometimes.
